asreview/webapp/src/ProjectComponents/SetupComponents/ModelComponents/ModelRequirement.js

```diff
@@ -24,7 +24,7 @@ const requirement = (requirement) => {
 
   return (
     <>
-      requires <code>{requirement}</code> to be installed.{" "}
```

Comment:
Maybe put a link to install instructions, or the actuall CLI input here?

Reply:
Thanks for this comment. I deprecate ModelRequirements.js and implement your suggestion in PR #1664. 

I make use of the link as I prevent CLI input suggestions because of the server version of ASReview.